### ISSUE TITLE
Fan speed control

### ISSUE THREAD

**Issue by psstapp** [anmalkov/minerkeeper#1]
Can you add a setfanspeed option per gpu or on all gpus.  This way the fans can also be monitored for failure.

**Comment by anmalkov** [comment]
Did you mean, that you want to receive a notification if current fan  speed is more or less than expected fan speed?

**Comment by psstapp** [comment]
Yes and also to set the fans speed at startup.  This would avoid loading extra apps to manage fan speeds.

for example {gpu1_fanspeed:60, gpu2_fanspeed:45, etc...}

and notify when fan speed is lower than 45%.

**Comment by anmalkov** [comment]
The notification about the low fan speed will be included in the next version.
However, I need to have more time for the actual control of the fan speed from MinerKeeper.

**Comment by psstapp** [comment]
Thanks for including the fan monitor in the next release.

Here are some thoughts.  

1. If you could include fan control then it would be great.  It would save a lot of problems.
2. Use case for OC Settings: Allow to OC the cards at run time with each run.  Right now, I use nVidia inspector to do this at logon or another overclocking tool.  However, I don't change these settings often.  So if you can include that it would reduce the amount of applications to be launched down to just MinerKeeper.
3. What gpu failed monitor:  Usually when there is a crash, a particular GPU is at fault.  If you can report back which GPU failed, it would make it a lot easier to diagnose the problem.

These are just a few suggestions.
